servers/update_api_gateway.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
The script that the CI/CD pipeline runs in the EC2 instance to deploy new changes